app/src/main/res/xml/grayscale_icon_map.xml

```diff
@@ -24,9 +24,11 @@
     <icon drawable="@drawable/app_manager" package="io.github.muntashirakon.AppManager" name="App Manager" />
     <icon drawable="@drawable/apple_music" package="com.apple.android.music" name="Apple Music" />
     <icon drawable="@drawable/application_inspector" package="com.ubqsoft.sec01" name="Application Inspector" />
+    <icon drawable="@drawable/ar_zone" package="com.samsung.android.arzone" name="AR Zone" />
     <icon drawable="@drawable/asana" package="com.asana.app" name="Asana" />
     <icon drawable="@drawable/audiofx" package="org.lineageos.audiofx" name="AudioFX" />
     <icon drawable="@drawable/aurora_droid" package="com.aurora.adroid" name="Aurora Droid" />
+    <icon drawable="@drawable/aurora_droid" package="sarangal.packagemanager" name="Package Manager" />
```

Comment:
Mind if I just ask why the package manager is linked to aurora droid? You could make an icon for the package manager with the settings icon in the middle instead of the aurora triangle

Reply:
Because this package manager used similar icon, so resused same, better I will create separate one , hence removed from current pr